test/unit/org/apache/cassandra/db/ColumnFamilyStoreTest.java

```diff
@@ -2233,4 +2237,34 @@ public void testRebuildSecondaryIndex() throws IOException
         
         PerRowSecondaryIndexTest.TestIndex.reset();
     }
+
+    @Test
+    public void testSnapshotWithoutFlushWithSecondaryIndexes() throws Exception
+    {
+        String keyspaceName = "Keyspace1";
+        String cfName= "Indexed1";
```

Comment:
Super nit, missed space char:
```suggestion
        String cfName = "Indexed1";
```